examples/stats_example.py

```
@@ -7,9 +7,10 @@
 
 import os
 from qwed_sdk import QWEDClient
+import os
```

Ruff (0.14.14)

[error] Redefinition of unused `os` from line 8: `os` redefined here

Remove definition: `os`

(F811)